styles/style.css

```diff
@@ -0,0 +1,219 @@
+* {
+  box-sizing: border-box;
+}
+
+.header-img {
+  background-image: url(../images/header-image.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 20em;
+}
+
+.logo {
+  max-width: 40%;
+  margin-left: 1em;
+  margin-top: 1em;
+}
+
+.tagline {
+  color: 	#81B642;
+  font-size: 2vw;
```

Comment:
I'd prefer you to keep to REMS or EMS for font sizes, it's a bit more standard at this point. If you really want to stick to vw thats's fine, but I'd encourage you to get used to REMS and EMS

Reply:
Is there a way to get the text to behave like vw using rems? I preferredd the vw method because it scales very nicely with the image when resizing windows. When using rems or ems, it always seems awkward.